content/v1/install.md

```diff
@@ -34,15 +34,15 @@ Install the Fermyon tap, which Homebrew tracks, updates, and installs Spin from:
 <!-- @selectiveCpy -->
 
 ```bash
-$ brew tap fermyon/tap
+$ brew tap spinframework/tap
 ```
 
 Install Spin:
 
 <!-- @selectiveCpy -->
 
 ```bash
-$ brew install fermyon/tap/spin
+$ brew install spinframework/tap/spin
 ```
 
 > Note: `brew install spin` will **not** install the Spin framework. Spin is accessed from the `fermyon` tap, as shown above.
```

Comment:
Oops, sorry, just noticed this one; can we update "`fermyon` tap" to "`spinframework` tap" here and in the other spots?

Reply:
@vdice this was my thought process around v1 and v2 https://github.com/spinframework/spin-docs/pull/37#discussion_r2017835535

Reply:
oh nice! Thanks! Yes.

Reply:
updated! Thanks for catching this!